HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+The high altitude balloon system offers a unique environment to develop skills and demonstrate prowess to RIT faculty and staff, potential SPEX members, and potential sponsors.
+A HAB bearing the mark of RIT SPEX is a declaration that we can confidently launch non-trivial payloads in a system of our making and have enough faith in our secure apparatus to link ourselves so. 
+HAB demands careful planning, intelligent utilization of resources, considerable multidisciplinary engineering and long-term teamwork. 
+A successful HAB is thus linked to a success-ready SPEX, whose members are now better prepared for any task. 
+HAB also serves as a unique testing platform for projects involving different temperatures, altitudes or pressures than those experienced normally. 
+This is particularly useful for lower-altitude rocket payloads, testing them for a fraction of the cost or risk. 
+HABs designed with a payload form resembling a cubesat or combination thereof have an even wider utilization. 
+HABs are also the easiest platform for testing of longer-range communication systems.
+
+The pursuit of HAB is the pursuit of maximizing scientific output using off-the-shelf components and present limitations of SPEX's human capital. 
+Progress here develops SPEX's knowledge base for all future missions, including SPEX's understanding of what can and cannot be achieved. 
+Presently proposed operations include rigorous pre-flight testing for diagnosis and correction of software and hardware components, post-flight analysis and amelioration and inculcation of this into the design cycle. 
+Potential payloads also include ``training'' for payloads utilizing machine learning. 
+All aspects SPEX greatly benefits from having.
+
+Another aspect to SPEX is the learning in the process, with documentation specified as a deliverable. 
+Also, by incorporating testing and training processes into the design and execution process, SPEX can iterate on more advanced software tools. 
+Specific focus on algorithmic improvisation may also draw in more talent, which in turn advances the scope of possible missions.
+
+Computer vision is field skirting the line between nascent and burgeoning. 
+As image processing tools advance, ever more useful information can be garnered from the same physical equipment. 
+Progress here automatically increases SPEX's potential to make news, capture imaginations and talent, and grow, for we will be at the relative forefront of technology---unlike projects aiming at aero-braking/recovery, novel communication or astronomical tracking, which are likely to lag behind by decades. 
+This advantage is both due to us maximizing the knowledge gleamed from cheap and readily available hardware, as well as the abundance of open source resources for computer vision. 
+While the entry requirements are non-zero, the acquisition of talent utilizing OpenCV or useful Python libraries broadens SPEX's options for other ventures, such as IREC's SDL Payload Challenge. 
```

Comment:
General proficiency itself goes a long way. Horizon detection for attitude or linking horizon detection with IMU information for altitude have both been proposed in the past.
Object recognition is another possible one, since the ~10000 feet altitude offers more to play with height(object recognition, selective deployment) than much higher launches. Arming of deployed payloads based on surroundings may be an interesting pursuit. For example: extremely high framerate capture based on a object detection.